src/main/scala/cromwell/engine/package.scala

```diff
@@ -19,7 +19,7 @@ package object engine {
   type WorkflowId = UUID
 
   sealed trait WorkflowState {
-    def isTerminal: Boolean
+    val isTerminal: Boolean
```

Comment:
I was trying to make it consistent, but your argument for why it was right the way it was before makes more sense than consistency.  :smile: